src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -96,25 +96,34 @@
 		function onEditOperationName(newOperationName) {
 			parameters.operationName = newOperationName;
 			updateURL();
-		}
+        }
 
-		function updateURL() {
-			var newSearch = '?' + Object.keys(parameters).filter(function (key) {
-				return Boolean(parameters[key]);
-			}).map(function (key) {
-				return encodeURIComponent(key) + '=' +
-					encodeURIComponent(parameters[key]);
-			}).join('&');
-			history.replaceState(null, null, newSearch);
-		}
+        function updateURL() {
+            var newSearch = '?' + Object.keys(parameters).filter(function (key) {
+                return Boolean(parameters[key]);
+            }).map(function (key) {
+                return encodeURIComponent(key) + '=' +
+                    encodeURIComponent(parameters[key]);
+            }).join('&');
+            history.replaceState(null, null, newSearch);
+        }
+
+        function stripGraphQLUiEndPointFromPathName(graphQlUiEndpoint) {
+            var currentPathName = window.location.pathname;
+            var regex = new RegExp(graphQlUiEndpoint + "\/*");
```

Comment:
Here the path name from the model variable is passed directly into a regular expression without encoding.  Also seems wrong.  Maybe this should not be using regexp at all and just perform a simple replace...